Formula/onelogin.rb

```diff
@@ -0,0 +1,35 @@
+class Onelogin < Formula
+  desc "CLI for Using OneLogin"
+  homepage "https://github.com/onelogin/onelogin"
+  url "https://github.com/onelogin/onelogin/archive/refs/tags/v0.1.14.tar.gz"
```

Comment:
Isn't there a `0.1.15` release?

Reply:
Yes. Will I have to update this file and submit a new PR for every patch?